src/util/select.h

```diff
@@ -58,25 +58,23 @@ class Select {
 private:
   Select()
     : max_fd( -1 )
-    /* These initializations are not used; they are just
-       here to appease -Weffc++. */
-    , all_fds( dummy_fd_set )
-    , read_fds( dummy_fd_set )
-    , empty_sigset( dummy_sigset )
-    , consecutive_polls( 0 )
-  {
+      /* These initializations are not used; they are just
+         here to appease -Weffc++. */
+      ,
```

Comment:
I think this comma should be on line 60, but that may be a clang-format self-stability issue?

Reply:
Also fixed by @keithw’s clang-format.